openmetadata-spec/src/main/resources/json/schema/entity/data/table.json

```diff
@@ -1002,6 +1002,10 @@
     "extension": {
       "description": "Entity extension data with custom attributes added to the entity.",
       "$ref": "../../type/basic.json#/definitions/entityExtension"
+    },
+    "sourceUrl": {
```

Comment:
actually thinking if, since we want to add the `sourceUrl` to all entities eventually, do we want to have a `url` basic type defined here? https://github.com/open-metadata/OpenMetadata/blob/main/openmetadata-spec/src/main/resources/json/schema/type/basic.json with `format: uri`?